fetch.bs

```diff
@@ -4924,10 +4954,14 @@ constructor must run these steps:
    to <var>method</var>.
   </ol>
 
+ <li><p>If <var>init</var>'s <code>signal</code> member is present, let <var>signal</var> be it.
+
  <li><p>Let <var>r</var> be a new {{Request}} object associated with <var>request</var> and
  a new associated {{Headers}} object whose <a for=Headers>guard</a>
  is "<code>request</code>".
 
+ <li><p>Set <var>r</var>'s <a for=Request>signal</a> to <var>signal</var>.
```

Comment:
Reading your additions it seems you could maybe set _r_'s signal directly each time? Or is _r_ not set up yet?